x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/authz/privilege/PrivilegeTests.java

```diff
@@ -469,27 +469,27 @@ public void testSlmPrivileges() {
     public void testDlmPrivileges() {
         assumeTrue("feature flag required", DataLifecycle.isEnabled());
         {
-            Predicate<String> predicate = IndexPrivilege.MANAGE_DLM.predicate();
+            Predicate<String> predicate = IndexPrivilege.MANAGE_DATA_STREAM_LIFECYCLE.predicate();
             // check indices actions
-            assertThat(predicate.test("indices:admin/dlm/explain"), is(true));
-            assertThat(predicate.test("indices:admin/dlm/get"), is(true));
-            assertThat(predicate.test("indices:admin/dlm/delete"), is(true));
-            assertThat(predicate.test("indices:admin/dlm/put"), is(true));
-            assertThat(predicate.test("indices:admin/dlm/brand_new_api"), is(true));
-            assertThat(predicate.test("indices:admin/dlm/brand_new_api"), is(true));
+            assertThat(predicate.test("indices:admin/data_stream/lifecycle/explain"), is(true));
+            assertThat(predicate.test("indices:admin/data_stream/lifecycle/get"), is(true));
+            assertThat(predicate.test("indices:admin/data_stream/lifecycle/delete"), is(true));
+            assertThat(predicate.test("indices:admin/data_stream/lifecycle/put"), is(true));
+            assertThat(predicate.test("indices:admin/data_stream/lifecycle/brand_new_api"), is(true));
+            assertThat(predicate.test("indices:admin/data_stream/lifecycle/brand_new_api"), is(true));
             // check non-dlm action
             assertThat(predicate.test("indices:admin/whatever"), is(false));
```

Comment:
Since we renamed the actions, let's also add:

```suggestion
            assertThat(predicate.test("indices:admin/whatever"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/create"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/delete"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/get"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/modify"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/promote"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/migrate"), is(false));
            assertThat(predicate.test("indices:admin/data_stream/brand_new_api"), is(false));
```

This is to make sure that the privilege does **not** grant access to data_stream actions.